docs/content/docs/recipes/audience-react.md

```diff
@@ -0,0 +1,404 @@
+    return (
+    // TODO 2: Return view components
+    );
+  }
+```
+
+1. Replace `TODO 1` with the following code. This code initializes local state variables that will be used within the application. `displayAudience` determines if we render the `AudienceDisplay` component or the `UserIdSelection` component (see `TODO 2`). `userId` contains the user identifier to connect to the container with and `containerId` is the container to load. `handleSelectUser` and `handleContainerNotFound` are passed in as callbacks to the two views and manage state transitions. `handleSelectUser` is called when attempting to create/load a container. `handleContainerNotFound` is called when creating/loading a container fails.
+
+Note, the values `userId` and `containerId` will come from a `UserIdSelection` component through the `handleSelectUser` function.
+
+```js
+  const [displayAudience, setDisplayAudience] = useState(false);
+  const [userId, setUserId] = useState();
+  const [containerId, setContainerId] = useState();
+
+  const handleSelectUser = useCallback((userId, containerId) => {
+    setDisplayAudience(true)
+    setUserId(userId);
+    setContainerId(containerId);
+  }, [displayAudience, userId, containerId]);
+
+  const handleContainerNotFound = useCallback(() => {
+    setDisplayAudience(false)
+  }, [setDisplayAudience]);
+```
+
+1. Replace `TODO 2` with the following code. As stated above, the `displayAudience` variable will determine if we render the `AudienceDisplay` component or the `UserIdSelection` component. Also, functions to update the state variables are passed into components as properties.
+
+```js
+  (displayAudience) ?
+  <AudienceDisplay userId={userId} containerId={containerId} onContainerNotFound={handleContainerNotFound}/> :
+  <UserIdSelection onSelectUser={handleSelectUser}/>
+```
+
+### Set up AudienceDisplay component
+
+1. Create and open a file `\src\AudienceDisplay.js` in the code editor. Add the following `import` statements:
+
+```js
+  import { useEffect, useState } from "react";
+  import { SharedMap } from "fluid-framework";
+  import { AzureClient } from "@fluidframework/azure-client";
+  import { InsecureTokenProvider } from "@fluidframework/test-client-utils";
+```
+
+Note that the objects imported from the Fluid Framework library are required for defining users and containers. In the following steps, `AzureClient` and `InsecureTokenProvider` will be used to configure the client service (see `TODO 1`) while the `SharedMap` will be used to configure a `containerSchema` needed to create a container (see `TODO 2`).
+
+1. Add the following functional components and helper functions:
+
+```js
+  const tryGetAudienceObject = async (userId, userName, containerId) => {
+    // TODO 1: Configure the client and container
+    // TODO 2: Configure the container
+    // TODO 3: Get the container and services from the Fluid service.
+    // TODO 4: Return the Audience object.
```

Comment:
I want to lean into your earlier point of not trying to teach everything. Let's just group this all together as 1 TODO with the description of reviewing https://fluidframework.com/docs/recipes/react/ and https://fluidframework.com/docs/build/containers/ if the developer want's to learn more about container creation.